luda-editor/rpc/src/data/sequence_update_action.rs

```diff
@@ -13,4 +13,8 @@ pub enum SequenceUpdateAction {
     DeleteCut {
         cut_id: Uuid,
     },
+    MoveCut {
+        cut_id: Uuid,
+        after_cut_id: Option<Uuid>,
```

Comment:
What's the `after` for?

Reply:
You mean, 'next of cut id'?

Reply:
Yes, same as [InsertCut](https://github.com/NamseEnt/namseent/pull/670/files/779f3dce0ae00e2c6317e7a264efb16e1065a0b3#diff-31e0f94759763b17a0cd73c478c4066e3dd0bb31e674cecbe4a98f84889267f1R51)